rules/how-to-share-a-file-folder-in-sharepoint/rule.md

```diff
@@ -10,29 +10,52 @@ authors:
   url: https://ssw.com.au/people/william-yin
 - title: Tiago Araujo
   url: https://ssw.com.au/people/tiago-araujo
-related: []
+related: 
+- avoid-using-share-functionality
+- do-you-know-not-to-send-attachments-in-emails
+- change-link-sharing-default-behaviour
 redirects:
 - do-you-know-how-to-share-a-file-folder-in-sharepoint
 
 ---
 
-You often need to share links to a file or folder in SharePoint. 
+You often need to share links to a file or folder in SharePoint.
 
 <!--endintro-->
 
+You may be tempted to share a link via the Share email invite. In doing so, the sharing permissions default to 'can view', which is fine if you are sharing the file externally. If you are sharing to someone who already has access to the file, it breaks permissions on the item you're sharing and creates a unique permission set for whoever you share it with. This means that a person who already had access to a file may find that using the shared link grants them different permissions. This makes it very difficult for SharePoint admins to maintain permissions!
+
+::: bad  
+![Figure: Bad example - Sharing URL via Share email invite](sharepoint-share-email-invite.png)  
+
+:::
+
+::: bad  
+![Figure: Bad example - Mulitple unique permission sets for a document library](sharepoint-file-unique-permissions.png)
```

Comment:
Remove this image as it is already in the other rule and the associated file

Reply:
Done